debian/rules

```diff
@@ -22,6 +22,10 @@ ifeq ($(DEB_HOST_ARCH), armel)
   extraopts += -DWITH_ATOMIC_OPS=OFF
 endif
 
+ifeq (,$(filter $(DEB_HOST_ARCH), s390 s390x))
```

Comment:
the conditional is supposed to be: "if DEB_HOST_ARCH equals "s390" or "s390x", then build with "-DWITH_RADOSGW_BEAST_FRONTEND=OFF"

I've never coded in this language, so I take your word for it that `ifneq` is correct in this case.